webapp/src/main/java/com/box/l10n/mojito/android/strings/XmlHelper.java

```diff
@@ -0,0 +1,293 @@
+            this.id = id;
+        }
+
+        @Override
+        public String getContent() {
+            return content;
+        }
+
+        @Override
+        public String getId() {
+            return id;
+        }
+    }
+
+    public enum PluralName {
+        zero,
+        one,
+        two,
+        few,
+        many,
+        other;
+
+        private static Map<String, Integer> map = IntStream
+                .range(0, PluralName.values().length).boxed()
+                .collect(Collectors.toMap(i -> PluralName.values()[i].toString(), Function.identity()));
+
+        private static int getIndex(String name) {
+            Integer index = map.get(name);
+            if (index == null) {
+                return Integer.MAX_VALUE;
+            } else {
+                return index;
+            }
+        }
+
+        public static void sortList(List<Item> itemList) {
+            itemList.sort(Comparator.comparingInt(i -> PluralName.getIndex(i.getPluralForm())));
+        }
+    }
+
+    public static class PluralXmlItem extends XmlItem {
+
+        private Map<PluralName, Item> itemMap = new HashMap<>();
+
+        private PluralXmlItem(XmlDocument xmlDocument, String comment, String name) {
+            super(xmlDocument,null, comment, name);
+        }
+
+        public PluralXmlItem addItem(PluralName name, String content, String id) {
+            itemMap.put(name, new SingleXmlItem(getDocument(), this, null, name.toString(), content, id));
+            return this;
+        }
+
+        public PluralXmlItem addItem(PluralName name, String content) {
+            return addItem(name, content, null);
+        }
+
+        public List<Item> getItems() {
+            List<Item> result = new ArrayList<>(itemMap.values());
+            PluralName.sortList(result);
+            return result;
+        }
+
+        @Override
+        public XmlDocument getDocument() {
+            return super.getDocument();
+        }
+    }
+
+    public static class XmlDocument {
+
+        private List<XmlItem> xmlItemList = new ArrayList<>();
+        private String nameSeparator = " _";
+
+        private String getPluralName(String parentName, String itemName) {
+            return parentName + nameSeparator + itemName;
+        }
+
+        public XmlDocument() {}
+
+        public XmlDocument(String nameSeparator) {
+            this.nameSeparator = nameSeparator;
+        }
+
+        public String getNameSeparator() {
+            return nameSeparator;
+        }
+
+        public XmlDocument addSingle(String comment, String name, String content, String id) {
+            xmlItemList.add(new SingleXmlItem(this, null, comment, name, content, id));
+            return this;
+        }
+
+        public XmlDocument addSingle(String comment, String name, String content) {
+            addSingle(comment, name, content, null);
+            return this;
+        }
+
+        public PluralXmlItem addPlural(String comment, String name) {
```

Comment:
this function is not needed. potentially  `public PluralXmlItem addPlural( String name, String content) {`
but just adding content doesn't make sense